end-to-end-tests/src/main/java/com/amazonaws/blox/integ/PersonalDevelopmentStack.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.integ;
+
+import com.amazonaws.blox.dataservicemodel.v1.client.DataService;
+import com.amazonaws.blox.integ.CloudFormationStacks.CfnStack;
+import java.util.Collections;
+import java.util.List;
+import lombok.Getter;
+import software.amazon.awssdk.services.cloudformation.CloudFormationClient;
+import software.amazon.awssdk.services.dynamodb.DynamoDBClient;
+import software.amazon.awssdk.services.ecs.ECSClient;
+import software.amazon.awssdk.services.ecs.model.DescribeTasksRequest;
+import software.amazon.awssdk.services.ecs.model.DesiredStatus;
+import software.amazon.awssdk.services.ecs.model.ListTasksRequest;
+import software.amazon.awssdk.services.ecs.model.StopTaskRequest;
+import software.amazon.awssdk.services.ecs.model.Task;
+import software.amazon.awssdk.services.lambda.LambdaAsyncClient;
+
+/** Facade over everything in an end-to-end Blox stack that's needed in tests. */
+public class PersonalDevelopmentStack implements TestStack {
+  private CloudFormationClient cloudformation = CloudFormationClient.create();
```

Comment:
why not inject all the clients using a spring config file?

Reply:
Let's come back to Spring later, there's not many dependencies right now, so it should be easy to wire things up with Spring once this gets any larger.